docker-compose-e2e.account.yaml

```diff
@@ -1,7 +1,7 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/compose-spec/compose-spec/master/schema/compose-spec.json
 x-common-environment: &common-environment
-  FREQUENCY_URL: ${FREQUENCY_URL:-ws://frequency:9944}
-  FREQUENCY_HTTP_URL: ${FREQUENCY_HTTP_URL:-http://localhost:9944}
+  FREQUENCY_BLOCKCHAIN_WS_URL: ${FREQUENCY_API_WS_URL:-ws://frequency:9944}
```

Comment:
In other places we are replacing `FREQUENCY_URL` with `FREQUENCY_API_WS_URL`. It is weird that in here we are using `FREQUENCY_BLOCKCHAIN_WS_URL`

Reply:
whoops! Good catch.